tests/test_dns_cli.py

```diff
@@ -183,7 +183,7 @@ def test_get_load_balancer_domain_and_configuration_no_clusters(capfd):
 @mock_ecs
 def test_get_load_balancer_domain_and_configuration_no_services(capfd):
     boto3.Session().client("ecs").create_cluster(
-        clusterName=f"{HYPHENATED_APPLICATION_NAME}-{ALPHANUMERIC_ENVIRONMENT_NAME}-{ALPHANUMERIC_SERVICE_NAME}"
+        clusterName=f"{HYPHENATED_APPLICATION_NAME}-{ALPHANUMERIC_ENVIRONMENT_NAME}-Cluster-c0PIlotiD3ntIF3r"
```

Comment:
Shall we create a constant for `Cluster-c0PIlotiD3ntIF3r` as well? I see it is used in another test as well.

Reply:
I extracted a couple more while I was at it. Less for duplication reasons, that readability.